modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/cxf/interceptors/EndpointImpl.java

```diff
@@ -58,4 +62,24 @@ public void increment()
       });
       return input + " " + cxfMessage.get(StringBuilder.class) + " " + counter.get();
    }
+   @WebMethod
+   public String echoException(String input) throws WebServiceException {
```

Comment:
Method input param, "input" is never used.  Doesn't it contain helpful information that can be included in the exception?  The current exception message has no helpful information for the user.